### PR TITLE
Add Mobile Support

### PR DESCRIPTION
**Pull request by libremente**
Added decorator to student_view for mobile compatibility.
Since `tox` creates a couple of files for testing purposes I think it's a good idea to remove those from the git indexing. 

@stvstnfrd @caesar2164 @caseylitton

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .coverage
 .tox/
+*.pyc
+*.egg-info/
```
Comment (caesar2164) [inline .gitignore:4]: I don't think these need the `*`s

Comment (libremente) [reply .gitignore:4]: you mean the  `egg-info/` folder? Probably not, I just followed the same done here:
https://github.com/Stanford-Online/xblock-free-text-response/blob/master/.gitignore
but I can remove it if necessary.

Comment (caesar2164) [reply .gitignore:4]: oh if there's a precedent, then nvmd!

**Comment by caesar2164** [comment]
👍

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**